datafusion/core/tests/physical_optimizer/push_down_filter.rs

```diff
@@ -154,29 +153,25 @@ impl FileSource for TestSource {
 
     fn try_pushdown_filters(
         &self,
-        mut fd: FilterDescription,
+        filters: &[Arc<dyn PhysicalExpr>],
```

Comment:
I would personally prefer keeping a struct as an argument as it
1. Is easier to document / explain what the structures mean via comments
2. it is clearer in the struct definition what types of operations (e.g. append only vs swap, etc) are done
3. It is easier to add new fields if needed

Basically,  something like

```rust
pub struct FilterDescription {
        filters: &[Arc<dyn PhysicalExpr>],
}
```
...

This is a preference thing and i can easily see alternative opinions

Reply:
Happy to do that 👍🏻